scripts/jira-automation.py

```diff
@@ -597,6 +761,17 @@ def main():
     if dry_run:
         logging.info("DRY_RUN is enabled")
 
+    parser = argparse.ArgumentParser(description='Snyk to Jira automation script')
+    parser.add_argument("-l", "--limit", type=str, help="The number of results to be returned by the snyk scan")
+    parser.add_argument("-v", "--version", type=str, help="The rest api version of snyk")
+    args = parser.parse_args()
+    snyk_api_result_limit = "100"
```

Comment:
you don't need this, just set the type and the default value in the argument definition